docs/index.html

```diff
@@ -36,540 +35,160 @@
 	<![endif]-->
 
 </head>
-<!-- BODY -->
-<!-- GRID-12 -->
+
 <body class="au-grid">
 
-	<div class="au-body">
-		<!-- SKIP-LINKS -->
-		<nav class="au-skip-link">
-			<a class="au-skip-link__link" href="#content">Skip to main content</a>
-			<a class="au-skip-link__link" href="#nav">Skip to main navigation</a>
-		</nav>
+	<!-- SKIP-LINKS -->
+	<nav class="au-skip-link au-body">
+		<a class="au-skip-link__link" href="#content">Skip to main content</a>
+		<a class="au-skip-link__link" href="#nav">Skip to main navigation</a>
+	</nav>
 
-		<!-- HEADER -->
-		<header class="au-header au-header--dark au-header--alt" role="banner">
-			<div class="container">
-				<div class="row">
-					<div class="col-sm-12">
-						<h1 class="au-header-heading">GOV.AU UI-Kit</h1>
+	<!-- HEADER -->
+	<header class="au-body au-header au-header--dark" role="banner">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-9">
+					<a class="au-header__brand" href="#">
+						<img class="au-header__brand-image" alt="Australian Government" src="./assets/img/header-logo-agov--light.png">
+						<div class="au-header__text">
+							<h1 class="au-header__heading">Example Site</h1>
+							<div class="au-header__subline">
+								An example of the system in use
+							</div>
+						</div>
+					</a>
+				</div>
+			</div>
+		</div>
+	</header>
 
-						<span class="au-header-subline">
-							Starter Pack
-						</span>
-					</div>
+	<!-- BANNER -->
+	<section class="au-body au-body--alt banner" id="content">
+		<div class="container">
+			<div class="row">
+				<div class="col-md-6">
+					<h3>Understand user needs. Research to develop a deep knowledge of the users and their context for using the service.</h3>
+					<br/>
+					<button class="au-btn au-btn--secondary banner__button">
+						Examples
+					</button>
+					<button class="au-btn banner__button">
+						Get Started
+					</button>
 				</div>
+				<div class="col-md-6">
+					<img class="au-responsive-media-img banner__image" src="http://placehold.it/900x300" alt="">
+				</div>
+			</div>
+		</div>
+	</section>
+
+	<!-- BREADCRUMBS -->
+	<div class="container au-body">
```

Comment:
Breadcrumbs don't need body as a dependency. So this class should be safe to remove.